tapdb/asset_minting.go

```diff
@@ -572,7 +572,8 @@ func fetchAssetSprouts(ctx context.Context, q PendingAssetStore,
 			if err != nil {
 				return nil, err
 			}
-			groupSig, err := schnorr.ParseSignature(sprout.GenesisSig)
+			// TODO(jhb): make generic for non-sig witnesses
+			groupSig, err := asset.ParseGroupSig(sprout.WitnessStack)
```

Comment:
Maybe we should update the `asset.GroupKey` to contain the parsed `TxWitness` instead of the sig? Then we can move this TODO to where we validate the sig?